client/utils/useViewport.ts

```diff
@@ -2,15 +2,22 @@ import { useEffect, useState } from 'react';
 
 export const useViewport = () => {
 	const [viewportWidth, setViewportWidth] = useState(null);
+	const [viewportHeight, setViewportHeight] = useState(null);
```

Comment:
Dang, sorry, I thought we already had this!

You can get rid of the `@ts-expect-error` comments that this generates by explicitly telling TypeScript what the valid values are for the state: `useState<null | number>(null)`. (Same with `viewportWidth`, if you don't mind)